packages/editor-ui/src/n8n-theme.scss

```diff
@@ -128,6 +128,30 @@
 	font-weight: bold;
 }
 
+.el-tabs__content {
```

Comment:
I meant tabs.scss before.. since our theme lives there.. but this is fine for now